modules/projects/client/views/view-project.client.view.html

```diff
@@ -25,11 +25,14 @@
         <button type="button" data-automation-id="button-project-edit" class="btn btn-default" ng-if="vm.canEdit"
           ui-sref="projectadmin.edit({projectId:vm.project.code})" title="Edit this page"><i class="fas fa-lg fa-pencil-alt"></i>
           Edit</button>
+        
```

Comment:
Let's remove these two comments.  Going forwarded, let's avoid including commented out chunks of code within the code base.

Reply:
Roger that about comments, @SeveChapmanBCDX. Will try to remember going forward. 🤝

I've removed that comment and pushed to the branch.